src/validators/minValue.js

```diff
@@ -0,0 +1,5 @@
+import {req, withParams} from './common'
+export default (min) => withParams(
+  { type: 'minValue', min },
+  value => !req(value) || value >= min
```

Comment:
There should be a cast to number, like `+value >= +min` to make it work like `between` validator.